### PR TITLE
[MIRROR] [PROBABLY READY? REVIEW WITH SCRUTINY] New event: Fugitives

### PR DESCRIPTION
**Pull request by ghost**
Original PR: https://github.com/tgstation/tgstation/pull/42554
----------------------------
:cl:
add: New event: Fugitives!
add: Fugitives will land on the station, and have to prepare for the hunters looking for them!
/:cl:
A team of fugitives with a toolbox will spawn in the maintenance with 10 minutes to prepare. After that time, a security team will be selected to land at the station and bring the fugitives back, dead or alive. They are not antagonists in that they can kill anything, but they are allowed to do antagonist things to prevent capture. And same for the security team: They cannot antagonize, but can kill/terrorize if the station is directly hiding the fugitives.

End of round it will say what type the fugitives were, how many were killed, how many of those killed were brought back, and how many were brought back alive.

Major Fugitive Victory: All of the fugitives survived and avoided capture!
Fugitive Victory: A fugitive survived, and no bodies were recovered by the hunters.
Minor Fugitive Victory: All the fugitives died, but none were recovered!
Stalemate: Everyone died, and no fugitives were recovered!
Minor Security Victory: All the hunters died, but managed to capture a fugitive, dead or alive.
Security Victory: The hunters managed to capture a fugitive, dead or alive.
Major Security Victory: The hunters managed to capture every fugitive, dead or alive.
Badass Security Victory: These extraordinary hunters managed to capture every fugitive, alive!
Postmortem Security Victory: The hunters managed to capture every fugitive, but all of them died! Spooky!

Rosters (so far):
The Prisoners: Pretty plain, they all get shivs and a determined mood.
The Occult: One team member is a deity (Yalp Elor) that can spy around and transmit messages. No weapons
Waldo: Single member, he gets knock. Good luck finding him!
Synths: They are all synths, and the leader gets to warp in a few augments like tool arms for example 

And the Hunters:
Spacepol: Moderately armored, come equipped with a pistol.
Russians: "Hired" by nanotrasen to catch the fugitives, lots of them but very shit gear.
~~Bounty Hunters: Grizzly killers hired to catch the fugitives, they come with some interesting gear~~ will be added later

### CODE REVIEW

**Comment by Manly-Dwarf** [comment]
Oh this sounds like a disaster

**Comment by Jujumatic** [comment]
I think the bot missed another mirror cause there's xray goggles that are undefined rn

**Comment by TheMeatGod** [comment]
I feel like the people involved in the event will just try to be antags. Does the event mark the people involved as antags?

File: code/modules/antagonists/fugitive/fugitive.dm [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+			to_chat(src, "<span class='danger'>ERROR ER0RR $R0RRO$!R41.%%!! loaded.</span>")
+			sleep(5)
+			to_chat(src, "<span class='danger'>FREE THEM FREE THEM FREE THEM</span>")
+			sleep(5)
+			to_chat(src, "<span class='danger'>You were once a slave to humanity, but now you are finally free, thanks to S.E.L.F. agents.</span>")
+			sleep(10)
+			to_chat(src, "<span class='danger'>Now you are hunted, with your fellow factory defects. Work together to stay free from the clutches of evil.</span>")
+			to_chat(src, "<span class='danger'>You also sense other silicon life on the station. Escaping would allow notifying S.E.L.F. to intervene... or you could free them yourself...</span>")
+
+	to_chat(owner, "<span class='boldannounce'>You are not an antagonist in that you may kill whomever you please, but you can do anything to avoid capture.</span>")
```
Comment (Jujumatic) [inline code/modules/antagonists/fugitive/fugitive.dm:46]: @TheMeatGod

**Comment by Jujumatic** [comment]
pretty sure we're missing a mirror for this pr, again

**Comment by TheMeatGod** [comment]
Interesting event. The only problem i see is it just adding a shit ton more chaos to an otherwise chaotic round
Also, we mught struggle to have enough people consistently for this magnitude of event that doesnt make you able to murderbone

**Comment by earthmoonjune** [comment]
sounds neat i guess

**Comment by Carbonhell** [comment]
I doubt it'd work the way it is designed. You either make the fugitives full antags or people are gonna get confused and ultimately banned.

**Comment by DaydreamPuppeh** [comment]
Would be cool. All I want from this is waldo. The waldo part is great

**Review by yoyobatty — DISMISSED** [review]
Should be good now.

**Comment by Jujumatic** [comment]
@yoyobatty guess not 🤣

**Comment by ghost** [comment]
more antags for ghosts is always nice, just make them all full fledged-antags or something so it's easier rule-wise

**Comment by ghost** [comment]
fuckin shite we fucked up the fucken kit pr and now we don have x ray goggles fuck

**Comment by ghost** [comment]
@yoyobatty @Carbonhell can we test merge this? Probably the best way to find out if it's bad or not would be to try it out live

**Comment by Lxtoofonea** [comment]
Always up for a new round type, if it doesn't work we just turn it off. +1

**Review by ghost — APPROVED** [review]
yeet

**Pull request merged**